### PR TITLE
Allows software directory with test vectors to be included in the con…

### PR DESCRIPTION
**Pull request by Grimgoodbye**
Kamyar,

Thanks so much for putting all the work you have into this. Suggestion: the mounted directory be LWC/ not LWC/hardware. This allows test vectors that are generated to be included with the config.ini file when they are in the software directory.

### CODE REVIEW

**Comment by kammoh** [comment]
Thank you for stepping up for help with this! I think having `../` in the docker mount point is actually not valid. I'd suggest we either change the definition of `$LWC_ROOT` to `LWC/` and update everything else accordingly, or introduce a new mount-point for `LWC/`.

File: hardware/dummy_lwc/src_rtl/configs/16MSconfig.ini [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[Generics]
```
[the remark this answers is not in the document]
Comment (Grimgoodbye) [reply hardware/dummy_lwc/src_rtl/configs/16MSconfig.ini:1]: I am all for doing it that way too but lets get this change in and then retro fit with that choice if we have time.

[the remark this answers is not in the document]
Comment (Grimgoodbye) [reply hardware/dummy_lwc/src_rtl/configs/16MSconfig.ini:1]: I am all for it. I just haven't looked at your config_parser.py that much.

[the remark this answers is not in the document]
Comment (Grimgoodbye) [reply hardware/dummy_lwc/src_rtl/configs/16MSconfig.ini:1]: In my next branch i have a script that is going through all of these configs already and seding things as required.

[the remark this answers is not in the document]
Comment (kammoh) [reply hardware/dummy_lwc/src_rtl/configs/16MSconfig.ini:1]: Sounds good!

**Pull request merged**